pkg/cli/environment.go

```diff
@@ -77,6 +79,7 @@ func New() *EnvSettings {
 		KubeAsUser:       os.Getenv("HELM_KUBEASUSER"),
 		KubeAsGroups:     envCSV("HELM_KUBEASGROUPS"),
 		KubeAPIServer:    os.Getenv("HELM_KUBEAPISERVER"),
+		KubeInsecure:     envBoolOr("HELM_KUBEAPISERVER_INSECURE", false),
```

Comment:
Why are we being inconsistent with the naming convention here? The feature flag is called `--kube-insecure-skip-tls-verify`, not `--kube-apiserver-insecure`.

Please follow the same convention as similar flags and re-name this as `HELM_KUBEINSECURE_SKIP_TLS_VERIFY`.

Reply:
My bad! the renaming is done now.